lib/libunw/unwind_riscv.c

```diff
@@ -0,0 +1,103 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*-
+ * Copyright (c) 2023 Andes Technology Corporation
+ * Copyright (c) 2015-2019 Linaro Limited
+ * Copyright (c) 2015 The FreeBSD Foundation
+ * All rights reserved.
+ *
+ * This software was developed by Semihalf under
+ * the sponsorship of the FreeBSD Foundation.
+ *
+ * Redistribution and use in source and binary forms, with or without
```

Comment:
the text starting from here could be omitted as it is standar dBSD-2-Clause terms.